packages/shared/lib/services/connection.service.ts

```diff
@@ -546,7 +546,7 @@ class ConnectionService {
                 environment_id,
                 deleted: false
             })
-            .update({ deleted: true, deleted_at: new Date() });
+            .update({ deleted: true, credentials: {}, credentials_iv: null, credentials_tag: null, deleted_at: new Date() });
```

Comment:
Any reason an empty object instead of null for credentials?

Reply:
`credentials` column is not nullable unfortunately